internal/storer/storer.go

```diff
@@ -19,6 +19,11 @@ const (
 	DefaultInterval = 15 * time.Minute
 )
 
+type Storer interface {
+	Set(key string, value interface{}) int64
+	Get(key string, valuePtr interface{}) (int64, error)
+}
```

Comment:
Storer was built to be used with SDK integrations. I'd rather not reuse it for other things as it is not super good. What about just a map instead?

Reply:
Done in 89b8e6004d8f746a9c2ce9cdaa28c96d7ca696e4.